components/list/masonry/src/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+
+class ListMasonry extends React.Component {
+  constructor (props) {
```

Comment:
Simpler without constructor:
`state = { columns: 1}`

Should be 1 a prop? At least a const?